### PR TITLE
Tests refactored

### PR DESCRIPTION
**Pull request by deleugpn**
- Added helper for factory;
- Added signIn helper to the TestCase;
- Reworked Group ViewComposer test to account for package Collection;
- Resolved problem with duplicity in default settings (#93);

### CODE REVIEW

File: src/ViewComposers/GroupComposer.php [hunk 1]
```diff
@@ -20,6 +20,6 @@ public function compose(View $view)
             ->with('descendants')
             ->get();
 
-        $view->with(compact('groups'));
+        $view->with('groups', $groups);
```
Comment (AndreasElia) [inline src/ViewComposers/GroupComposer.php]: Please revert this change

Comment (deleugpn) [inline src/ViewComposers/GroupComposer.php]: It breaks the tests

Comment (Dylan-DPC) [inline src/ViewComposers/GroupComposer.php]: It shouldn't be `with(compact(...` but just `compact`
